test/tools/HttpsListener/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Net;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.AspNetCore.Server.Kestrel.Https;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+
+namespace HttpsListener
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            if (args.Count() != 3)
+            {
+                System.Console.WriteLine("Required: <CertificatePath> <CertificatePassword> <PortNumber>");  
```

Comment:
@iSazonov  The password is either supplied at invocation, or is hard coded. In theory, We may need to run multiple listeners with multiple certificates that might not have the password set to `password`. I thought supplying it at listener invocation was better than hard coding it in the listener.